### ISSUE TITLE
These are the errors I get when I try to package the file... 

### ISSUE THREAD

**Issue by neatville** [kivy/buildozer#222]
buildozer android debug deploy run....

http://pastebin.com/Wd0Gxq3B

**Comment by lcddrm** [comment]
The last line of the error log tells you what to do:

`Exception: Unable to find capture version in ./main.py`
```python
(looking for `__version__ = ['"](.*)['"]`)
```
At the beginning of your main.py, put this:
```python
__version__ = "1.0"
```
The version number in the string could of course be any version

**Comment by neatville** [comment]
I added that to that one, and it actually did end up installing it, but the program would crash without notice each time I tried running it. 
Then, I went to a different example Kivy program to see if that one would work, 3Drendering to be more specific. It gave me the same kinds of errors I've experienced before when trying to debug & deploy using buildozer - Cython errors. It's up to the latest version, and that's what's recommended on the Kivy site. I've heard some say in issues from months ago that this version doesn't work with Kivy, so last night I had tried it with .21 or something then tried it with that... still errors. 

There was apparently a Kivy update recently, I was hoping that might solve all of the problems, but it didn't.
